lxqt-globalkeys-config-version.cmake wrong package version

cmake problems when mixing lxqt-globalkeys 0.14.2 with lxqt 0.14.1 packages..
Found be building lxqt-runner 0.14.1
```
CMake Error at /usr/local/share/cmake/Modules/CMakeFindDependencyMacro.cmake:47 (find_package):
  Could not find a configuration file for package "lxqt-globalkeys" that is
  compatible with requested version "0.14.2".

  The following configuration files were considered but not accepted:

    /usr/local/share/cmake/lxqt-globalkeys/lxqt-globalkeys-config.cmake, version: 0.14.1

Call Stack (most recent call first):
  /usr/local/share/cmake/lxqt-globalkeys-ui/lxqt-globalkeys-ui-config.cmake:43 (find_dependency)
  CMakeLists.txt:39 (find_package)
```

Fix is 
```diff
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -72,7 +72,7 @@ configure_package_config_file(
 #************************************************
 write_basic_package_version_file(
     "${CMAKE_BINARY_DIR}/${LXQT_GLOBALKEYS_CMAKE_NAME}-config-version.cmake"
-    VERSION ${LXQT_VERSION}
+    VERSION ${LXQT_GLOBALKEYS_VERSION}
     COMPATIBILITY AnyNewerVersion
 )
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,13 +72,13 @@ configure_package_config_file(
 #************************************************
 write_basic_package_version_file(
     "${CMAKE_BINARY_DIR}/${LXQT_GLOBALKEYS_CMAKE_NAME}-config-version.cmake"
-    VERSION ${LXQT_VERSION}
+    VERSION ${LXQT_GLOBALKEYS_VERSION}
     COMPATIBILITY AnyNewerVersion
 )
 
 write_basic_package_version_file(
     "${CMAKE_BINARY_DIR}/${LXQT_GLOBALKEYS_UI_CMAKE_NAME}-config-version.cmake"
-    VERSION ${LXQT_VERSION}
+    VERSION ${LXQT_GLOBLKEYS_VERSION}
     COMPATIBILITY AnyNewerVersion
 )
 #************************************************

